pages/c/_cluster/fleet/index.vue

```diff
@@ -118,29 +119,92 @@ export default {
       });
     },
     badgeClass(area, row) {
+      // classes are defined in the themes SASS files...
       switch (area) {
       case 'clusters':
         if (row.clusterInfo?.ready === row.clusterInfo?.total && row.clusterInfo?.ready) {
-          return 'green-badge';
+          return 'bg-success';
         }
 
-        return 'red-badge';
+        return 'bg-warning badge-class-default';
       case 'bundles':
-        if (row.bundlesReady?.length === row.bundles.length && row.bundlesReady?.length) {
-          return 'green-badge';
+        if (row.bundles?.length && row.bundles?.every(bundle => bundle.state?.toLowerCase() === STATES_ENUM.ACTIVE)) {
+          return `bg-${ STATES[STATES_ENUM.ACTIVE].color || 'unknown' }`;
+        }
+        if (row.bundles?.length && row.bundles?.some(bundle => bundle.state?.toLowerCase() === STATES_ENUM.ERR_APPLIED)) {
+          return `bg-${ STATES[STATES_ENUM.ERR_APPLIED].color || 'unknown' }`;
+        }
+        if (row.bundles?.length && row.bundles?.some(bundle => bundle.state?.toLowerCase() === STATES_ENUM.NOT_READY)) {
+          return `bg-${ STATES[STATES_ENUM.NOT_READY].color || 'unknown' }`;
         }
 
-        return 'red-badge';
+        if (row.bundlesReady?.length === row.bundles?.length && row.bundlesReady && row.bundles?.length) {
+          return 'bg-success';
+        }
+
+        return 'bg-warning badge-class-default';
       case 'resources':
+        if (row.status?.resources?.length && row.status?.resources?.every(resource => resource.state?.toLowerCase() === STATES_ENUM.ACTIVE)) {
+          return `bg-${ STATES[STATES_ENUM.ACTIVE].color || 'unknown' }`;
+        }
+        if (row.status?.resources?.length && row.status?.resources?.some(resource => resource.state?.toLowerCase() === STATES_ENUM.ERR_APPLIED)) {
+          return `bg-${ STATES[STATES_ENUM.ERR_APPLIED].color || 'unknown' }`;
+        }
+        if (row.status?.resources?.length && row.status?.resources?.some(resource => resource.state?.toLowerCase() === STATES_ENUM.NOT_READY)) {
+          return `bg-${ STATES[STATES_ENUM.NOT_READY].color || 'unknown' }`;
```

Comment:
`STATES[STATES_ENUM.unknown]`

Reply:
mapped to the UNKNOWN state (so that we would have a default color) but also added `bg-unmapped-state` so that we can easily catch problems while inspecting the UI from the browser, if needed